kayenta-sql/src/main/resources/db/changelog/202303091551-create-canary-archive-table.yml

```diff
@@ -0,0 +1,29 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-canary-archive-table
+      author: alexey.bedonik
+      changes:
+        - createTable:
+            tableName: canary_archive
+            columns:
+              - column:
+                  name: id
+                  type: VARCHAR(255)
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: content
+                  type: TEXT
```

Comment:
Ideally, this column type should be `JSON` for MySQL-style databases and `JSONB` for PostgreSQL-style databases. This makes it much easier to handle data migrations as an admin over time, and on the JSONB side of things, it's far more space efficient. Same for the other tables with a generic column for holding JSON data.

Reply:
I have an example in https://github.com/spinnaker/kork/blob/ec5c486656cd2ba28c50cc1c11f6e83879b50a4f/kork-credentials-sql/src/main/resources/db/changelog/20230217-accounts-body-typefix.yml on how I was able to finagle Liquibase to support this. If you default to a `JSON` column type with a followup changelog entry to update PostgreSQL databases to use `JSONB`, that seems to work. Note that these columns work effectively like `TEXT` columns when doing normal queries, so there's nothing to change in your entities.